dpgen2/op/run_dp_train.py

```diff
@@ -128,9 +127,7 @@ def _make_train_command_old(
             checkpoint,
             train_script_name,
         ]
-    elif (
-        do_init_model or finetune_mode == "train-init"
-    ) and not init_model_with_finetune:
+    elif do_init_model and not init_model_with_finetune:
```

Comment:
it seems that `_make_train_command_old` is not used anymore, we may simply remove the function.

Reply:
`_make_train_command_old` has been removed.